data/brands/shop/bakery.json

```diff
@@ -898,6 +898,18 @@
         "shop": "bakery"
       }
     },
+    {
+      "displayName": "Rotiboy",
+      "id": "rotiboy-2acef0",
+      "locationSet": {"include": ["ms"]},
```

Comment:
from https://en.wikipedia.org/wiki/Rotiboy
> The company has more than 100 outlets in Malaysia, Indonesia, Thailand, South Korea, Saudi Arabia and the United Arab Emirates, as well planning to enter China market.[2]

Can you add these other countries to the list also?